package.json

```diff
@@ -71,7 +71,7 @@
     "@guardian/src-button": "^1.8.0",
     "@guardian/src-foundations": "^1.9.0",
     "@guardian/src-svgs": "^1.8.0",
-    "@guardian/types": "^0.3.0",
```

Comment:
Is this a new syntax for GitHub packages? Does it always fetch master or the latest tag release? might we want to pin to a specific version?

Reply:
By default I believe it points to the latest version of master. You can further specific which commit/tag to use by using an `#`, which you can use for versioning. See more here: https://docs.npmjs.com/files/package.json#git-urls-as-dependencies

Reply:
The version pinning is handled in [`package-lock.json`](https://github.com/guardian/apps-rendering/pull/522/files#diff-32607347f8126e6534ebc7ebaec4853dR4571); it specifies the repo and specific commit on `master` as the version.

Reply:
Would an `npm i` overwrite the lock file to a later version?

Reply:
I don't think so... if it did it would defeat the point of a lockfile wouldn't it?

Reply:
hmm yeah, but how would the lockfile know what version to use? It might be more explicit in the package.json too

Reply:
If you don't specify a version when you first add it to the project it picks the latest `master` commit and locks the package to that. Alternatively you can specify a specific commit when you first add it as @ripecosta mentioned above.